Makefile

```
@@ -80,6 +80,10 @@ test: envtest
 test-integrationv2: envtest
 	KUBEBUILDER_ASSETS="$(shell setup-envtest use $(ENVTEST_K8S_VERSION) -p path)" go test ./test/... -coverprofile cover.out
 
+.PHONY: testv2
+testv2:
+	go test ./pkg/controller.v2/... ./pkg/runtime.v2/... ./pkg/webhook.v2/... ./pkg/util.v2/... -coverprofile cover.out
```

```suggestion
	go test ./pkg/.. -coverprofile cover.out
```

This can at least help future people if a new folder is added and they don't need to wonder why there was not coverage for their package.

---

This is not available due to v1 testing codes.
After we remove v1 codes, we can migrate to the approach.

Additionally, we proposed some automation here: https://github.com/kubeflow/training-operator/pull/2248#discussion_r1752320236

But, the approach is rejected.